attributes/default.rb

```diff
@@ -90,4 +90,9 @@
 default['cloud']['queue_config']['capacity']             = "10000"
 default['cloud']['queue_config']['max_sample_per_send']  = "5000"
 default['cloud']['queue_config']['batch_send_deadline']  = "60s"
-default['cloud']['metrics']['port']                      = "9096"
\ No newline at end of file
+default['cloud']['metrics']['port']                      = "9096"
```

Comment:
You are missing at the top of attributes/default.rb (add in line 7)
include_attribute "kube-hops"

If you don't include this, the attributes in lines 95-98 might not find the correct prometheus/root_dir attribute.

Reply:
Attributes were moved from kube-hops here, no longer relevant.